pkg/scheduler/scheduler.go

```diff
@@ -276,7 +276,7 @@ func New(client clientset.Interface,
 		extenders = append(extenders, ignorableExtenders...)
 	}
 
-	registry := frameworkplugins.NewInTreeRegistry(frameworkplugins.WithIgnoredResources(ignoredResources...))
+	registry := frameworkplugins.NewInTreeRegistry(frameworkplugins.WithIgnoredResources(ignoredResources))
```

Comment:
what about ignored resources from policies?

Reply:
Good catch, I overlooked this when removing `FitArgs`. I moved the registry instantiation [into the `create()` call ](https://github.com/kubernetes/kubernetes/pull/88768/commits/bab6972a33fd82dab943f4d8fa3e62d26be5fbf5#diff-ec926856bfdbbcaf5e9dfce784909d51R151)shared by provider/policy, after processing the extenders.